### PR TITLE
Add Bindings for Classic and XAddress Checks

### PR DESCRIPTION
**Pull request by keefertaylor**
Add bindings for `isXAddress()` and `isClassicAddress()` in Swift. Unit test implementations. 

Tests taken from the [implementation in JavaScript](https://github.com/xpring-eng/xpring-common-js/pull/50)

### CODE REVIEW

File: XpringKit/JavaScript/JavaScriptUtils.swift [hunk 1]
```diff
@@ -6,27 +6,32 @@ internal class JavaScriptUtils {
...
+    public static let isValidXAddress = "isValidXAddress"
+    public static let tag = "tag"
     public static let utils = "Utils"
 	}
 
 	/// Native javaScript functions wrapped by this class.
-	private let isValidAddressFunction: JSValue
-    private let encodeXAddressFunction: JSValue
-    private let decodeXAddressFunction: JSValue
+  private let encodeXAddressFunction: JSValue
```
Comment (intelliot) [inline XpringKit/JavaScript/JavaScriptUtils.swift]: indentation looks weird? or maybe it's just the way github is showing it

File: XpringKit/JavaScript/JavaScriptUtils.swift [hunk 2]
```diff
@@ -40,40 +45,62 @@ internal class JavaScriptUtils {
...
-      let result = encodeXAddressFunction.call(withArguments: arguments)!
-      return result.isUndefined ? nil : result.toString()
+  /// Encode the given classic address and tag into an x-address.
+  ///
+  /// - SeeAlso: https://xrpaddress.info/
+  ///
+  /// - Parameters:
+  ///   -  classicAddress: A classic address to encode.
+  ///   - tag: An optional tag to encode. Defaults to nil.
+  /// - Returns: A new x-address if inputs were valid, otherwise undefined.
```
Comment (intelliot) [inline XpringKit/JavaScript/JavaScriptUtils.swift]: nit: usually we are capitalizing the X in X-address

File: XpringKit/JavaScript/JavaScriptUtils.swift (hunk 1 above)
Comment (keefertaylor) [reply XpringKit/JavaScript/JavaScriptUtils.swift]: Sorry about that.  Fixed. 

I think that at some point I had xcode set to tabs vs spaces and it messed up the formatting. Outside of this CL, I'll look at whether we can catch that with `swiftlint` or if I can do a global file reformat.

File: XpringKit/JavaScript/JavaScriptUtils.swift (hunk 2 above)
Comment (keefertaylor) [reply XpringKit/JavaScript/JavaScriptUtils.swift]: Done

**Pull request merged**